zebra-consensus/src/transaction/check.rs

```diff
@@ -99,14 +99,20 @@ pub fn has_enough_orchard_flags(tx: &Transaction) -> Result<(), TransactionError
 
 /// Check that a coinbase transaction has no PrevOut inputs, JoinSplits, or spends.
 ///
-/// A coinbase transaction MUST NOT have any transparent inputs, JoinSplit descriptions,
-/// or Spend descriptions.
+/// # Consensus
 ///
-/// In a version 5 coinbase transaction, the enableSpendsOrchard flag MUST be 0.
+/// >  A coinbase transaction MUST NOT have any transparent inputs with non-null prevout fields,
+/// > JoinSplit descriptions, or Spend descriptions.
+///
+/// > [NU5 onward] In a version 5 coinbase transaction, the enableSpendsOrchard flag MUST be 0.
 ///
 /// This check only counts `PrevOut` transparent inputs.
 ///
-/// https://zips.z.cash/protocol/protocol.pdf#txnencodingandconsensus
+/// > [Pre-Heartwood] A coinbase transaction also MUST NOT have any Output descriptions.
+///
+/// Zebra will not validate this last rule explicitly because we checkpoint until Canopy blocks.
```

Comment:
Nit: 'will' makes it sound like Zebra's validation is incomplete.

```suggestion
/// Zebra does not validate this last rule explicitly because we checkpoint until Canopy activation.
```